ansible/roles/sshkey-host-access/README.md

```diff
@@ -0,0 +1,34 @@
+sshkey-host-access
+=========
+
+Create and move keypairs to new instances for access through Web Shells.
+
+Role Variables
+--------------
+
+| Variable                | Required | Default | Choices                   | Comments                                 |
+|-------------------------|----------|---------|---------------------------|------------------------------------------|
+| ATMOUSERNAME            | yes      |         |                           | Username of user getting the keys        |
+| LOCAL_PUBKEY_DIR        | yes      | `/tmp/` |                           | the directory to transfer the public key locally |
+| KEY_LOC_PATH            | yes      |`/var/lib/gateone/users/{{ ATMOUSERNAME }}/.ssh`|| the directory to transfer the public key |
+| KEY_LOC_OWNER           | yes      | root    | "root", "tomcat7"         | owner of new keys                        |
+| KEY_LOC_GROUP           | yes      | root    | "root", "tomcat7"         | group of new keys                        |
+| KEY_LOC_PREFIX          | yes      | ""      | "_guac", ""               | prefix to name key for specific server   |
+| target_host             | yes      | shell   | "shell", "guac_server"    | hostname for delegated tasks             |
+| FULL_PUBKEY_PATH        | yes      |         |                           | simplifies long string of other variables|
+| FULL_PRIVKEY_PATH       | yes      |         |                           | simplifies long string of other variables|
+| LOCAL_FULL_PUBKEY_PATH  | yes      |         |                           | simplifies long string of other variables|
```

Comment:
I submitted some changes to illustrate what i mean. 
https://github.com/calvinmclean/atmosphere-ansible/pull/5